invokeai/frontend/web/src/features/modelManagerV2/subpanels/AddModelPanel/ScanFolder/ScanFolderForm.tsx

```diff
@@ -17,11 +17,13 @@ export const ScanModelsForm = () => {
   const [_scanFolder, { isLoading, data }] = useLazyScanFolderQuery();
 
   const scanFolder = useCallback(async () => {
-    _scanFolder({ scan_path: scanPath }).catch((error) => {
-      if (error) {
-        setErrorMessage(error.data.detail);
-      }
-    });
+    _scanFolder({ scan_path: scanPath })
+      .unwrap()
```

Comment:
Unwrapped this because I noticed the error wasn't getting caught otherwise

Reply:
Good catch!